release.json

```diff
@@ -11,7 +11,7 @@
         "WINDOWS_DDNPM_SHASUM": "4ca09e6a029dc1db2f2313b6a1ebf770ee3e733dcf500c770e9cd154490d0435"
     },
     "nightly-a7": {
-        "INTEGRATIONS_CORE_VERSION": "master",
+        "INTEGRATIONS_CORE_VERSION": "alex/snmp_telemetry",
```

Comment:
Note for reviewers: This will be removed before merging to master. This is currently used to make it easier to benchmark snmp core check vs snmp python check.